GOVERNANCE.md

```diff
@@ -56,10 +56,10 @@ The current team members are:
 - Mauro Stettler — [@replay](https://github.com/replay) ([Grafana Labs](https://grafana.com/))
 - Nick Pillitteri — [@56quarters](https://github.com/56quarters) ([Grafana Labs](https://grafana.com/))
 - Oleg Zaytsev — [@colega](https://github.com/colega) ([Grafana Labs](https://grafana.com/))
+- Patrick Oyarzun - [@Logiraptor](https://github.com/Logiraptor) ([Grafana Labs](https://grafana.com/))
```

Comment:
just for consistency

```suggestion
- Patrick Oyarzun - [@Logiraptor](https://github.com/logiraptor) ([Grafana Labs](https://grafana.com/))
```

Reply:
GitHub usernames are caseinsensitive, but looking at Patrick profile I think the actual setting with the uppercase L:
![Screenshot 2022-08-10 at 10 39 21](https://user-images.githubusercontent.com/1701904/183856196-633273f2-c4d9-426c-8ad8-df6937cc147a.png)

Reply:
In comparison mine is lowercase:
![Screenshot 2022-08-10 at 10 40 17](https://user-images.githubusercontent.com/1701904/183856353-b6fafb1e-0cc5-4edc-b525-1032c44d751b.png)

Reply:
TIL